flocker/cli/script.py

```diff
@@ -250,9 +250,49 @@ def _changestate_on_nodes(self, deployment, deployment_config,
         return DeferredList(results)
 
 
+@flocker_standard_options
+class CLIOptions(Options):
+    """
+    Command line options for ``flocker`` CLI.
+
+    """
+    longdesc = """flocker does nothing (yet).
+
+    """
+
+    synopsis = ("Usage: flocker [OPTIONS] "
+                "\n"
+                "If you have any issues or feedback, you can talk to us: "
+                "http://docs.clusterhq.com/en/latest/gettinginvolved/"
+                "contributing.html#talk-to-us")
+
+
+@implementer(ICommandLineScript)
+class CLIScript(object):
+    """
+    A command-line script to interact with a cluster via the API.
+    """
+    def main(self, reactor, options):
+        """
+        See :py:meth:`ICommandLineScript.main` for parameter documentation.
+
+        :return: A ``Deferred`` which fires when the deployment is complete or
```

Comment:
This is something that's eventually replacing `flocker-deploy` (as far as I understand the plan), but will also be developed while we're still keeping the old deploy script. I think this comment string remains accurate so far as the intention of what the new CLI will be doing, just how it's doing deployments will be different (interacting with an API rather than YAML over SSH). So the comment string may need to change, but again I also don't know yet. For the moment, as a stub script, a "successful deployment" is actually just returning a succeeded deferred.